homeassistant/components/totalconnect/.translations/en.json

```diff
@@ -0,0 +1,20 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Account already configured"
+        },
+        "error": {
+            "login": "Login error: please check your username & password"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "password": "Password",
+                    "username": "Username"
+                },
+                "title": "Fill in your Total Connect login information"
```

Comment:
There is also a description field that might better suite this information

Reply:
Simplified this text.